waku/v0_test.go

```diff
@@ -339,6 +339,7 @@ func checkTestStatus() {
 		}
 	}
 
+	// TODO Should this be removed? debug is always false and has no method of updating, this code will never be reached
 	if debugMode {
 		if cntPrev != cnt {
 			fmt.Printf(" %v \t number of nodes that have received all msgs: %d, number of peers per node: %v \n",
```

Comment:
Should this be removed? Debug is always false and has not method of updating, this code will never be reached.

Reply:
Yes, sounds like it can be removed :)

Reply:
Agreed.